kiwixbuild/dependencies/all_dependencies.py

```diff
@@ -12,6 +12,9 @@ class AllBaseDependencies(Dependency):
     class Builder(NoopBuilder):
         @classmethod
         def get_dependencies(cls, platformInfo, allDeps):
+            if platformInfo.build == "wasm":
+                return ['zlib', 'lzma', 'zstd', 'icu4c', 'xapian-core', 'zim-testing-suite']
```

Comment:
I am surprised that `zim-testing-suite` stays as a dependency for `wasm`. Do tests run under `wasm`?

Reply:
Indeed. No need for `zim-testing-suite`